apps/test-app/frontend/src/components/tree-widget/stateless/Tree.tsx

```diff
@@ -161,6 +181,32 @@ export function StatelessTreeWidget(props: Omit<TreeWidgetProps, "rulesetId">) {
   );
 }
 
+interface FormatterSetterDropdownProps {
+  shouldUseCustomFormatter: boolean;
+  setShouldUseCustomFormatter: (value: React.SetStateAction<boolean>) => void;
+}
+
+function FormatterSetterDropdown(props: FormatterSetterDropdownProps) {
+  const dropdownMenuItems = (close: () => void) => [
+    <MenuItem
+      key={1}
+      onClick={() => {
+        props.setShouldUseCustomFormatter((state) => !state);
+        close();
+      }}
+    >
+      {props.shouldUseCustomFormatter ? "Show custom formatter" : "Show default formatter"}
```

Comment:
This looks backwards. When `shouldUseCustomFormatter` is `true` custom formatter is used and button has text - `"Show custom formatter"`

Reply:
Changed